packages/swingset-runner/src/main.js

```diff
@@ -303,7 +303,8 @@ export async function main() {
         logStats = true;
         break;
       case '--logtag':
-        logTag = argv.shift();
+        nextArg = argv.shift();
+        nextArg && (logTag = nextArg);
```

Comment:
If `--logtag ''`, is invalid, it should be rejected with an error rather than silently ignored.